packages/flutter_tools/lib/src/convert.dart

```diff
@@ -44,14 +43,16 @@ class Utf8Codec extends Encoding {
 
 const Encoding utf8 = Utf8Codec();
 
-class Utf8Decoder extends cnv.Utf8Decoder {
-  const Utf8Decoder({this.reportErrors = true}) : super(allowMalformed: true);
+class Utf8Decoder extends Converter<List<int>, String> {
+  Utf8Decoder({this.reportErrors = true})
+      : decoder = const cnv.Utf8Decoder(allowMalformed: true);
```

Comment:
The `decoder` here can be a (private) static const variable, no need for an instance variable that's always the same value anyway.

Also no need to remove the `const` from the constructor, it should still work as `const` (either way).

Here's my version: https://github.com/flutter/flutter/pull/123211